packages/utils/types/date.d.ts

```diff
@@ -10,4 +10,18 @@ declare module '@endpass/utils/date' {
   export function addToDate(date: DayJsDate, value: number, unit: string): Date;
 
   export function getWeek(date: Date): number;
+
+  export function toEqualLocalTime(date: DayJsDate): Date;
```

Comment:
Dayjs? We can pass Date, timestamp...

Reply:
`type DayJsDate = Date | object | string | number;`